README.md

```diff
@@ -31,35 +31,26 @@ The ```main``` branch of this repository contains samples configured for the lat
 
 The *ArcGIS Runtime SDK Samples app* has a *Target SDK* version of *13.0*, meaning that it can run on devices with *iOS 13.0* or newer.
 
-## Building samples using installed SDK
+## Building Samples Using Swift Package Manager
 
 1. **Fork** and then **clone** the repository
-    > Make sure to use the "recursive" option to ensure you get the **ArcGIS Runtime Toolkit** submodule
-    >
-    >`git clone --recursive [URL to Git repo]`
-    >
-    > If you've already cloned the repo without the submodule, you can load the submodule using 
-    >
-    >`git submodule update --init`
-1. **Install** the ArcGIS Runtime SDK for iOS to a central location on your Mac as described [here](https://developers.arcgis.com/ios/get-started)
-1. **Open** the `arcgis-ios-sdk-samples.xcodeproj` **project** file
 1. **Run** the `arcgis-ios-sdk-samples` app target
-    > If you get the error message saying _"This Copy Files build phase contains a reference to a missing file 'ArcGISToolkit.framework'"_, you probably didn't clone the repo to include it's submodule. See Step 1 above.
 
-## Building samples with CocoaPods
+> To add the Swift Packages to your project, consult the [documentation](https://github.com/Esri/arcgis-runtime-toolkit-ios/#swift-package-manager) in the ArcGIS Runtime Toolkit repo.
+
+## Building Samples Using CocoaPods
 
 1. **Fork** and then **clone** the repository
 1. **Install** the ArcGIS Runtime SDK for iOS by running the `pod install` command in the folder where you cloned this repository
 1. **Open** the `arcgis-ios-sdk-samples.xcworkspace` **workspace** file
-1. **Select** the `arcgis-ios-sdk-samples` project node, go to the `Build Phases` tab, and **delete** the phases for `Embed Frameworks`. (This phase conflicts with CocoaPods and is only required when using the installed SDK as described in the previous section)
-1. **Select** the `ArcGISToolkit.xcodeproj` project node and **delete** it. (this project dependency conflicts with CocoaPods and is only required when using the installed SDK as described in the previous section)
+1. **Select** the `arcgis-ios-sdk-samples` project node, go to the `Swift Packages` tab, and **delete** the `arcgis-runtime-toolkit-ios` package. (This Swift Package conflicts with CocoaPods and is only required when using the Swift Package Manager as described in the previous section)
```

Comment:
Maybe this should be done after the fork and clone to avoid the collision.

Reply:
@zkline101 I thought about too, but if it is not technically necessary, then these steps are a better flow for the user. Otherwise, they'll have to open the xcode project....delete sp dependency... close the project.... run pod install....open workspace..... too...much...faff

Reply:
😂  I agree with both... cannot decide which way to go. Should I just revert?

Reply:
Test If the current sequence works, and if it does, keep it.

Reply:
Yeah, @dg0yal is right it kinda messes up the flow. Can you revert please?